src/lib/components/sections/navbar.svelte

```diff
@@ -205,6 +205,9 @@
 
           a {
             font-size: var(--size-md);
+            display: block;
+            padding: 30px 900px;
```

Comment:
is possible to avoid hard coded values here or no?

Reply:
I will try I might be able to

Reply:
A cheap (imo) workaround is to just turn the navbar into a sidebar when the screen is narrow. That way, it can sit on the right with a fixed width.